sdk/servicebus/service-bus/test/internal/unit/atomXml.spec.ts

```diff
@@ -110,24 +122,30 @@ describe("ATOM Serializers", () => {
     });
 
     it("should properly serialize when using valid inputs and serializer", async function () {
-      const request = new WebResource();
-      request.body = { lockDuration: "PT3M", maxSizeInMegabytes: "2048" };
+      const request = createPipelineRequest({
+        url: "https://someurl",
+      });
+      const requestObject = buildQueueOptions({ lockDuration: "PT3M", maxSizeInMegabytes: 2048 });
```

Comment:
Why did you have to make this change? ...using `buildQueueOptions`

Reply:
I made a change to not rely on internal logic in `executeAtomXmlOperation()` to stringify body to XML
85d7737d09f1c3cce38bf3b2f5e62e98c4275db0